lms/tasks/organization.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+
+from lms.services import OrganizationService
+from lms.tasks.celery import app
+
+
+@app.task
+def generate_usage_report(
+    organization_id: int, tag: str, since: str, until: str
```

Comment:
Nothing fancy here, just take the params needed as json serializable parameters and call the service method.